locales/ja/php.json

```
@@ -28,7 +28,7 @@
     "digits_between": ":Attributeは、:min桁から:max桁にしてください。",
     "dimensions": ":Attributeの画像サイズが無効です",
     "distinct": ":Attributeの値が重複しています。",
-    "doesnt_start_with": "The :attribute may not start with one of the following: :values.",
+    "doesnt_start_with": ":attributeは次のうちいずれかで始まってはいけません。: :values",
```

Is it correct that the sentence ends with a full stop before a colon with a choice of options?

Maybe it should be like this? Or am I wrong?

```suggestion
    "doesnt_start_with": ":attributeは次のうちいずれかで始まってはいけません: :values。",
```

---

@andrey-helldar 

I am not 100% sure which one is correct.
it might be my sentence. it might be your sentence. It could be both.

is `:values` something like this?
`A, B, C`

---

Given the name of the key, under the hood the validator will explicitly check with the `in_array` function, so the final sentence in English might look like this:

> The :attribute may not start with one of the following: :values.
> The first name may not start with one of the following: foo, bar, baz.

When first name is `fooable`, for example.

I don't know the Japanese grammar rules, but in many countries the full stop is placed at the end of a sentence, not before the list of meanings.

That's why I checked if it's correct.

---

We stick to the writing format, but the end goal is grammatically correct sentences.